packages/express-test-utils/src/index.ts

```diff
@@ -1,6 +1,6 @@
-import { Server } from 'node:http';
+import { type Server } from 'node:http';
 
-import * as express from 'express';
+import type * as express from 'express';
```

Comment:
I wonder if it makes sense to import the type `Express` here instead? Nit.

Reply:
Done in ad94972.